extraneous semicolon in LGTM starter

### Describe the bug

The LGTM starter seems to introduce a compile error

### Expected behavior

It creates and builds a project.

### Actual behavior

```
[ERROR] Failed to execute goal org.apache.maven.plugins:maven-compiler-plugin:3.14.1:testCompile (default-testCompile) on project leptonflow-camel-example: Compilation failure
[ERROR] ./src/test/java/ch/rabe/labs/SimpleTest.java:[3,62] extraneous semicolon
[ERROR] 
[ERROR] -> [Help 1]
[ERROR] 
```

### How to Reproduce?

```yaml
---
type: specs.openrewrite.org/v1beta/recipe
name: ch.rabe.labs.AddQuarkusLGTM
displayName: Add Quarkus LGTM dependency
recipeList:
  - org.openrewrite.maven.AddDependency:
      groupId: io.quarkus
      artifactId: quarkus-observability-devservices-lgtm
      scope: provided
      version: 3.30.1
```

### Output of `uname -a` or `ver`

_No response_

### Output of `java -version`

openjdk version "21.0.8" 2025-07-15 LTS OpenJDK Runtime Environment Temurin-21.0.8+9 (build 21.0.8+9-LTS) OpenJDK 64-Bit Server VM Temurin-21.0.8+9 (build 21.0.8+9-LTS, mixed mode, sharing)

### Quarkus version or git rev

3.30.1

### Build tool (ie. output of `mvnw --version` or `gradlew --version`)

_No response_

### Additional information

I'm going to bed now, I'll provide a fix tomorrow unless someone beats me to removing `;;` in the codepaths from [this search](https://github.com/search?q=repo%3Aquarkusio%2Fquarkus+SimpleTest.java+%3B%3B&type=code)

--- a/devtools/project-core-extension-codestarts/src/main/resources/codestarts/quarkus/extension-codestarts/lgtm-codestart/java/src/test/java/org/acme/SimpleTest.java
+++ b/devtools/project-core-extension-codestarts/src/main/resources/codestarts/quarkus/extension-codestarts/lgtm-codestart/java/src/test/java/org/acme/SimpleTest.java
@@ -1,6 +1,6 @@
 package org.acme;
 
-import org.eclipse.microprofile.config.inject.ConfigProperty;;
+import org.eclipse.microprofile.config.inject.ConfigProperty;
 import org.jboss.logging.Logger;
 import org.junit.jupiter.api.Test;
 import org.junit.jupiter.api.condition.DisabledOnOs;

